data/css/modules/layout/_sidebar.scss

```diff
@@ -18,20 +18,14 @@
             }
 
             &:hover {
-                // background: transparentize(white, 1 - 0.05);
```

Comment:
It would be better to aggregate these changes to the commit where you commented these line.